docs/platform/custom-cloud-credentials/customer-managed-byoc.md

```diff
@@ -0,0 +1,249 @@
+# Deploying Customer Managed Bring Your Own Cluster (BYOC) Mode
+!!! note
```

Comment:
add this to the top of the page 
`import Note from "@site/src/components/Note";`

wrap the note in `<note></note>`